Cmdline/CKAN-cmdline.csproj

```diff
@@ -92,7 +92,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\Core\CKAN-core.csproj">
-      <Project>{3B9AEA22-FA3B-4E43-9283-EABDD81CF271}</Project>
+      <Project>{EBE1FF06-FAB5-4EFD-A65D-7085C5281919}</Project>
```

Comment:
This is another case of me not knowing enough about this, but does the project UUID *need* to change? Maybe it's necessary, but if it's not, my preference would be to keep it the same.

Reply:
I’m not sure. Visual Studio did that automatically, so I will look into why that happened.

Reply:
So the reason why it changed is because I removed the line specifying the project's GUID, causing Visual Studio to automatically assign it a new GUID. I removed the line because it is optional in the new csproj format. However, removing one line from the csproj is certainly not sufficient reason to change the GUID. :smile